packages/notification-center/package.json

```diff
@@ -77,6 +77,7 @@
     "axios": "^0.26.1",
     "chroma-js": "^2.4.2",
     "date-fns": "^2.29.1",
+    "lodash.clonedeep": "^4.5.0",
```

Comment:
`lodash.clonedeep` is used instead of `structuredClone` which is only available rn in the browser (web API)